server/plugin.go

```diff
@@ -99,10 +100,31 @@ func (p *Plugin) OnConfigurationChange() error {
 }
 
 func (p *Plugin) OnActivate() error {
-	conf := p.getConfig()
-	user, appErr := p.API.GetUserByUsername(conf.UserName)
-	if appErr != nil {
-		return errors.WithMessage(appErr, fmt.Sprintf("OnActivate: unable to find user: %s", conf.UserName))
+	botUserID, err := p.Helpers.EnsureBot(&model.Bot{
+		Username:    botUserName,
+		DisplayName: botDisplayName,
+		Description: botDescription,
+	})
+	if err != nil {
+		return errors.Wrap(err, "failed to ensure bot account")
+	}
+
+	p.updateConfig(func(conf *config) {
+		conf.botUserID = botUserID
+	})
+
+	bundlePath, err := p.API.GetBundlePath()
+	if err != nil {
+		return errors.Wrap(err, "couldn't get bundle path")
+	}
+
+	profileImage, err := ioutil.ReadFile(filepath.Join(bundlePath, "assets", "profile.png"))
```

Comment:
Should put this in the `/public`? @hanzei would want it I think, and would be right. We didn't do it before only to be 5.10-compatible.

Reply:
No, this should not be in `/public`. `/public` is used to serve files via http. This file just gets read and then send via RPC. There is not need to expose this to the public.

This is fine the way it is.